### PR TITLE
Add Spark 2.4.1 link to doc

### PR DESCRIPTION
**Pull request by dbtsai**
*Make sure that you generate site HTML with `jekyll build`, and include the changes to the HTML in your pull request also. See README.md for more information. Please remove this message.*

### CODE REVIEW

**Comment by dbtsai** [comment]
@srowen @dongjoon-hyun forget to update the spark 2.4.1 link in the previous PR. Can you take a second look? Thanks!

File: js/downloads.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.0+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
-addRelease("2.4.0", new Date("11/02/2018"), packagesV9, true);
+addRelease("2.4.1", new Date("04/01/2019"), packagesV9, true);
```
Comment (dongjoon-hyun) [inline js/downloads.js]: Ur, it should be March 31.
- https://lists.apache.org/thread.html/fbec33f7e601f25b8c950ec42a3ee287bd8623d973ea3a2c8d7f2fbb@%3Cdev.spark.apache.org%3E

File: site/js/downloads.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.0+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
-addRelease("2.4.0", new Date("11/02/2018"), packagesV9, true);
+addRelease("2.4.1", new Date("04/01/2019"), packagesV9, true);
```
Comment (dongjoon-hyun) [inline site/js/downloads.js]: ditto.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Pull request merged**